ygen/ir.go

```diff
@@ -98,12 +98,24 @@ type LangMapperBaseSetup interface {
 	//  - identityrefs within typedefs
 	// to be resolved to the corresponding type that is to be used in
 	// the IR.
-	SetEnumSet(*enumSet)
+	setEnumSet(*enumSet)
 
-	// SetSchemaTree is used to supply a copy of the YANG schema tree to
+	// setSchemaTree is used to supply a copy of the YANG schema tree to
 	// the mapped such that leaves of type leafref can be resolved to
 	// their target leaves.
-	SetSchemaTree(*schemaTree)
+	setSchemaTree(*schemaTree)
+
+	// SetupEnumSet is intended to be called by unit tests in order to set up the
```

Comment:
Should we call this `InjectEnumSet` or something similar to show that it is being used specifically for that purpose? `Setup...` seems to indicate to me that this is something that would be required for non-testing purposes.

Reply:
Done.